scripts/fast-reboot

```diff
@@ -332,6 +341,21 @@ function reboot_pre_check()
     fi
 }
 
+function load_aboot_secureboot_kernel() {
+    local next_image="$IMAGE_PATH/sonic.swi"
+    echo "Loading next image from $next_image"
+    (
+        cd /tmp
+        unzip -qp "$next_image" boot0 | \
+            swipath=$next_image kexec=true testonly=true ENV_EXTRA_CMDLINE="$BOOT_OPTIONS" bash -
```

Comment:
>testonly [](start = 43, length = 8)

why `testonly`?
The name seems weird.

Reply:
This is a setting that existed for a long time in boot0 and that basically stops after `kexec -l` and right before `kexec -e`
I agree that the setting name is confusing, I will add another one called `loadonly=true` that will have the exact same behavior.
I originally just reused the old setting name as we can't easily remove it for legacy reasons.
I will make the change in https://github.com/Azure/sonic-buildimage/pull/4994